src/modules/endpoint/DatabaseConnector.js

```diff
@@ -187,6 +201,40 @@ class DatabaseConnector {
     await this.writeFile(path, JSON.stringify(languages))
   }
 
+  async storeCities (cities: Array<CityModel>) {
+    const jsonModels = cities.map((city: CityModel): ContentCityJsonType => ({
```

Comment:
Couldn't you just call `JSON.stringify()` on the `Array<CityModel>`? I think the `map` here is not necessary.

Reply:
We should explicitly use the `map` here imo. We do not expect the JSON files to change if we change the `CityModel`